### ISSUE TITLE
Error:fail:

### ISSUE THREAD

**Issue by klausrose** [tducasse/go-instabot#96]
Hey, I'm getting this error every time I try to run the instabot. I have replaced the config file many times already. I pasted the config folder in my user directory. Am I missing something? 
![image](https://user-images.githubusercontent.com/103354212/162607283-3a28add6-8afb-4497-8a28-daf077d2cd81.png)

**Comment by tducasse** [comment]
Mmm I'm sorry this is very little information for me to help... Are you sure the config file inside the config folder is valid and you've added the right values correctly?

**Comment by klausrose** [comment]
Oh, happy to talk with you!

I downloaded the code via git hub (zip file), the installed the instabot
through CMD as you said. I couldn't use the "get" command so I had to use
install (you know, I replaced the get with install, that's all). Then I
edited the config file with a text editor (not Notepad++) and pasted the
config folder in my User folder (C:\users\User). But only the config file.

I'm really a noob in all of this. I may have made a mistake. :(
